README.md

```diff
@@ -1,17 +1,23 @@
 
 # supported [![CI](https://github.com/stefanpenner/supported/workflows/CI/badge.svg)](https://github.com/stefanpenner/supported/actions/workflows/ci.yml)
 
-
-
-
 ## Usage
 
-
 ```sh
 npx supported <path/to/node_module>
 npx supported <[array/of/node_modules]>
 ```
 
+#### Optional Flags
+
+The `--date` flag enables a form of limited time travel, and attempts to run
+the tools internal date calculations based on a specified date, rather then the
+current date.
+
+* `--date="March 31, 2011"` runs the tool as if it was March 31, 2021
+* `--date="-3 weeks"` runs the tool as if it was 3 weeks ago
+* `--date="3 weeks"` runs the tool as if it was 3 weeks from now
```

Comment:
It will be great if we can list/link to all types of available unit types. 
I found that the `prase-duration` library didn't provide a heading so that we can refer. I created a [PR](https://github.com/jkroso/parse-duration/pull/31) in the repo for the same.

Reply:
PR is merged now.
We can use https://github.com/jkroso/parse-duration#available-unit-types-are

Reply:
great idea